### PR TITLE
Design task/task 2

### PR DESCRIPTION
**Pull request by ShaneKainos**
Duplicated files from Design Task 3, band folder files are the majority of what needs to be reviewed.

### CODE REVIEW

**Review by jessicamcmillen — APPROVED** [review]

File: kjourney/src/app/view-tabs/view-tabs.component.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<div class="container">
+  <ngb-tabset>
+
+    <ngb-tab title="Job Family Diagrams">
```
Comment (MetalGearFridge) [inline kjourney/src/app/view-tabs/view-tabs.component.html]: what is a job family diagram

**Review by MetalGearFridge — APPROVED** [review]
yeah looks fine like idk nothing significant here

File: kjourney/src/app/app.module.ts [hunk 2]
```diff
@@ -8,12 +8,22 @@ import { AppRoutingModule } from './app-routing.module';
...
+import { View3Component } from './view3/view3.component';
+import { BandComponent } from './band/band.component';
 
 @NgModule({
   declarations: [
     AppComponent,
     MainComponent,
-    LoginComponent
+    LoginComponent,
+    ViewTabsComponent,
```
Comment (Conormc98) [inline kjourney/src/app/app.module.ts]: Remove ViewTabsComponent, View2Component and View3Component

File: kjourney/src/app/view-tabs/view-tabs.component.html (hunk 1 above)
Comment (ShaneKainos) [reply kjourney/src/app/view-tabs/view-tabs.component.html]: A diagram of Job Families

File: kjourney/src/app/view-tabs/view-tabs.component.html [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<div class="container">
```
Comment (Conormc98) [inline kjourney/src/app/view-tabs/view-tabs.component.html]: Delete the entirety of the View-Tabs, View2 and View3 components

**Review by Conormc98 — CHANGES_REQUESTED** [review]

**Review by Conormc98 — APPROVED** [review]

**Pull request merged**